rocky/tools/management/commands/setup_dev_account.py

```diff
@@ -46,6 +46,7 @@ def handle(self, *args, **options):
                 "add_organizationmember",
                 "change_organization",
                 "change_organizationmember",
+                "recalculate_bits",
```

Comment:
The permission also needs to be added to the list of permissions in tools/models.py